src/audio/component.c

```diff
@@ -160,6 +160,17 @@ void comp_get_copy_limits(struct comp_buffer __sparse_cache *source,
 	cl->sink_bytes = cl->frames * cl->sink_frame_bytes;
 }
 
+void comp_get_copy_limits_frame_aligned(struct comp_buffer __sparse_cache *source,
+					struct comp_buffer __sparse_cache *sink,
+					struct comp_copy_limits *cl)
```

Comment:
Is it possible to make the first two arguments `const struct`? If not why not? This shows the difference between IN and OUT arguments nicely and it may catch bugs.